From 5aab1c265b3d6d0339453bce590707fb2ba25a3d Mon Sep 17 00:00:00 2001
From: Sergei Reznikov <diver@gelios.net>
Date: Mon, 16 Mar 2015 20:44:40 +0300
Subject: Haiku: comment out subprocess functions

* We lack spawn.h header so we can't use posix-spawn from subprocess-posix.c
* subprocess-haiku.c probably needs to be implemented

diff --git a/player/command.c b/player/command.c
index b89f7c0..71058c3 100644
--- a/player/command.c
+++ b/player/command.c
@@ -4014,7 +4014,7 @@ static void overlay_uninit(struct MPContext *mpctx)
         overlay_remove(mpctx, id);
     osd_set_external2(mpctx->osd, NULL);
 }
-
+/*
 struct subprocess_args {
     struct mp_log *log;
     char **args;
@@ -4047,7 +4047,7 @@ static void subprocess_detached(struct mp_log *log, char **args)
     if (pthread_create(&thread, NULL, run_subprocess, p))
         talloc_free(p);
 }
-
+*/
 struct cycle_counter {
     char **args;
     int counter;
@@ -4631,7 +4631,7 @@ int run_command(MPContext *mpctx, mp_cmd_t *cmd)
         char *args[MP_CMD_MAX_ARGS + 1] = {0};
         for (int n = 0; n < cmd->nargs; n++)
             args[n] = cmd->args[n].v.s;
-        subprocess_detached(mpctx->log, args);
+//        subprocess_detached(mpctx->log, args);
         break;
     }
 
-- 
2.2.2

